cpp/src/arrow/compute/api_aggregate.h

```diff
@@ -201,30 +201,42 @@ Result<Datum> MinMax(
 /// \brief Test whether any element in a boolean array evaluates to true.
 ///
 /// This function returns true if any of the elements in the array evaluates
-/// to true and false otherwise. Null values are skipped.
+/// to true and false otherwise. Null values are ignored by default.
+/// If null values are taken into account by setting ScalarAggregateOptions
+/// parameter skip_nulls = false then kleen logic is applied.
```

Comment:
```suggestion
/// parameter skip_nulls = false then Kleene logic is used.
```

Maybe also add something like "see OrKleene for details on Kleene logic"

Reply:
I think it KleeneOr. Added.